miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -178,6 +188,82 @@ export.set_account_item
     # => [R', V]
 end
 
+#! Returns VALUE located under specified KEY in map in specified account storage slot. 
+#! Panics if 
+#! - the index is out of bounds.
+#! - the requested storage slot type is not map
+#!
+#! Stack: [index, KEY, ...]
+#! Output: [VALUE]
```

Comment:
**Request**: This is being called with a syscall. So I think the Output here should be:

```
#! Output: [VALUE, 0]
```

Otherwise I _think_ we may push a `0` at 16th position of the caller's stack.

Reply:
Interesting. We should alsways return with 16 elements, right? And the elements 0,1,2,3 should be VALUE. At least if I understand this correctly. https://0xpolygonmiden.github.io/miden-vm/user_docs/assembly/execution_contexts.html

```
When the called procedure returns, the following happens:

Execution moves back to the context from which the procedure was invoked.
Stack depth is set to its original depth. Before the stack depth is reset, the VM checks if the current stack depth is exactly 16, and fails otherwise.
```

Reply:
For some reason, it doesn't work if I push another 0 at the end. 

I get an invalid stack depth return of 17